pytest_mongodb/plugin.py

```diff
@@ -109,7 +109,7 @@ def load_fixture(db, collection, path, file_format):
             _cache[path] = docs = loader(fp)
 
     for document in docs:
-        db[collection].insert(document)
+        db[collection].insert_one(document)
```

Comment:
The problem is, that this would make the code unusable for projects using older versions of mongomock or pymongo. I guess one has to dynamically figure out which method is available like so
```python
try:
    insert_one = getattr(db[collection], 'insert_one')
except AttributeError:
    insert_one = getattr(db[collection], 'insert')

for document in docs:
    insert_method(document)
```

Reply:
revised